package.json

```diff
@@ -489,9 +508,13 @@
       },
       {
         "command": "vscode-docker.container.start.azurecli",
-        "title": "Azure CLI",
+        "title": "Run Azure CLI",
```

Comment:
I'm kind of confused by this command. It feels like generic Azure tooling that _happens_ to leverage a container, rather than container-specific tooling. Is it really worth putting in the title bar?

Also - doesn't this overlap with the cloud shell commands? In both cases you don't need the Azure cli installed. Maybe we just need to make _those_ commands more obvious somehow
![screen shot 2018-09-24 at 1 32 02 pm](https://user-images.githubusercontent.com/11282622/45977391-4c72f200-bffe-11e8-9c64-35c4f4c30b1c.png)

Maybe another button next to the filter button?
![screen shot 2018-09-24 at 1 45 27 pm](https://user-images.githubusercontent.com/11282622/45978060-1cc4e980-c000-11e8-81c4-32f96ad634d6.png)

Reply:
Good question.  @chrisdias @fiveisprime Is there a reason to have both "Run Azure CLI" and "Open Bash in Cloud Shell"?

Reply:
The CLI container will mount in the file system, right? That (along with not provisioning time) makes it much more usable than the cloud shell.

Reply:
@fiveisprime so should this be in the azure account extension so that more people benefit? Again - this doesn't feel like docker-specific tooling. It just _happens_ to use docker

Reply:
Correct.  It looks like (from quick docs perusal) that cloud shell is limited to drives in Azure storage.